src/java/picard/analysis/GcBiasMetricsCollector.java

```diff
@@ -235,6 +235,12 @@ else if (group.equals("Library")) {
                     summary.WINDOW_SIZE = scanWindowSize;
                     summary.TOTAL_CLUSTERS = totalClusters;
                     summary.ALIGNED_READS = totalAlignedReads;
+                    summary.GC_NC_0_20 = calculateGcNormCoverage(meanReadsPerWindow, readsByGc, 0, 19);
```

Comment:
The names of the metrics should reflect the actual values used despite the less pretty result.